assets/javascripts/initializers/disable-sort.js

```diff
@@ -0,0 +1,20 @@
+import discourseComputed from "discourse-common/utils/decorators";
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "disable-sort",
+  initialize() {
+    withPluginApi("0.8", (api) => {
+      api.modifyClass("component:topic-list", {
+        @discourseComputed("category")
+        sortable(category) {
+          let disableSort = true;
```

Comment:
@frank3manuel a bug spotted by @keegangeorge – sorting topics doesn't work in e.g. /latest

I believe that was supposed to be `false` by default?

Reply:
If this hasn't been fixed yet, I'll fix it now.

Reply:
PR for the fix is up. #334